src/tasks/add-triage-issue-if-new.ts

```diff
@@ -7,7 +7,8 @@ export class AddTriageIssueIfNew implements IssueHandler {
 
   public execute(issueInfo: IssueInfo, actions: Actions, notifier: Notifier): void {
     // already a status, ignore it
-    if (issueInfo.isClosed() || issueInfo.hasStatus() || issueInfo.repositoryName() !== "che" || issueInfo.milestone() != "" || issueInfo.hasSeverity()) {
+    if (issueInfo.isClosed() || issueInfo.hasStatus() || issueInfo.repositoryName() !== "che" || issueInfo.milestone() != "" || issueInfo.hasSeverity()
+       || issueInfo.hasMatchingLabel("automated")) {
```

Comment:
```suggestion
       || issueInfo. hasLabel("automated")) {
```

the label will be `automated` correct ? no prefix/suffix ?

Reply:
Yes, in this case hasLabel is a better fit?

Reply:
yes